### PR TITLE
Calculator component implemented

### PR DESCRIPTION
**Pull request by Ndaboom**
- I added the files calculate.js and operate.js into a logic/ directory in your project.
- Create a component ***calculator*** for rendering the calculator and all its operations performed.

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import React, { useState } from 'react';
+import calculate from '../logic/calculate';
+
+function Calculator(props) {
+  const [state, setState] = useState({
+    total: 0,
+    next: 0,
+    operation: '',
+  });
+
```
Comment (mahbubul14) [inline src/components/calculator.js]: - [ ] The Calculator.js component should be class-based as per project requirements. Kindly work on it. :+1:

File: src/App.js [hunk 2]
```diff
@@ -1,29 +1,12 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import Calculator from './components/calculator';
+import './index.css';
 
 function App() {
   return (
```
Comment (mahbubul14) [inline src/App.js]: - [ ] The App.js component should be class-based. :+1:

File: src/components/calculator.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+      <div className="touch-pad">
+        <button onClick={clickHandler} type="button">AC</button>
+        <button onClick={clickHandler} type="button">+/-</button>
+        <button onClick={clickHandler} type="button">%</button>
+        <button onClick={clickHandler} className="orange" type="button">÷</button>
+
+        <button onClick={clickHandler} type="button">7</button>
+        <button onClick={clickHandler} type="button">8</button>
+        <button onClick={clickHandler} type="button">9</button>
+        <button onClick={clickHandler} className="orange" type="button">*</button>
```
Comment (mahbubul14) [inline src/components/calculator.js]: - [ ] The multiply (*) button is not working properly. Please fix it. :+1:

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
**Hi @Ndaboom,**

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

* N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by arslanbisharat — COMMENTED** [review]
Hi @Ndaboom, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has been reviewed for another project. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by sumairq — APPROVED** [review]
### Hi @Ndaboom  , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**